### PR TITLE
Add Windows CROSSTOOL example

### PR DESCRIPTION
**Pull request by meteorcloudy**

### CODE REVIEW

File: toolchains/windows-crosstool/LICENSE [hunk 1]
```diff
@@ -0,0 +1,202 @@
+
```
Comment (dslomov) [inline toolchains/windows-crosstool/LICENSE]: You do not need your own LICENSE, the whole examples repo is already covered

File: toolchains/windows-crosstool/CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# How to Contribute
```
Comment (dslomov) [inline toolchains/windows-crosstool/CONTRIBUTING.md]: Not needed, the repo already has this

File: toolchains/windows-crosstool/README.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
+# Bazel Windows MSVC toolchain
+A standalone Windows CROSSTOOL for Bazel
+(disclaimer: This is not an official Google product.)
```
Comment (dslomov) [inline toolchains/windows-crosstool/README.md]: Drop this disclaimer

File: toolchains/windows-crosstool/LICENSE (hunk 1 above)
Comment (meteorcloudy) [inline toolchains/windows-crosstool/LICENSE]: Done.

File: toolchains/windows-crosstool/CONTRIBUTING.md (hunk 2 above)
Comment (meteorcloudy) [inline toolchains/windows-crosstool/CONTRIBUTING.md]: Done

File: toolchains/windows-crosstool/README.md (hunk 3 above)
Comment (meteorcloudy) [inline toolchains/windows-crosstool/README.md]: Done

**Comment by laurentlb** [comment]
What's the state of this PR?

**Comment by meteorcloudy** [comment]
I think it's no longer needed.

**Pull request closed without merging**